hibernate-core/src/main/java/org/hibernate/cache/spi/access/package.html

```diff
@@ -33,10 +33,10 @@
 </p>
 <p>
     Note that, for an <i>asynchronous</i> cache, cache invalidation must be a two step process (lock->unlock or
-    lock->afterUpdate), since this is the only way to guarantee consistency with the database for a nontransactional
+    lock->afterUpdate), since this is the only way to guarantee consistency with the database for a non-transactional
     cache implementation. For a <i>synchronous</i> cache, cache invalidation is a single step process (evict or update).
-    Hence, these contracts ({@link org.hibernate.cache.spi.access.EntityRegionAccessStrategy} and
-    {@link org.hibernate.cache.spi.access.CollectionRegionAccessStrategy}) define a three step process to cater for both
+    Hence, these contracts ({@link org.hibernate.cache.spi.access.EntityDataAccess} and
+    {@link org.hibernate.cache.spi.access.CollectionDataAccess}) define a three step process to cater for both
     models (see the individual contracts for details).
```

Comment:
Needs input. But it seems my understanding is not wrong.

Reply:
Seems fine.